sdk/communication/azure-communication-identity/tests/recordings/test_communication_identity_client.test_get_token_for_teams_user_with_invalid_token.yaml

```diff
@@ -13,13 +13,13 @@ interactions:
       Content-Type:
       - application/json
       User-Agent:
-      - azsdk-python-communication-identity/1.1.0b1 Python/3.9.7 (macOS-11.6-x86_64-i386-64bit)
+      - azsdk-python-communication-identity/1.2.0 Python/3.7.12 (Darwin-21.3.0-x86_64-i386-64bit)
       x-ms-date:
-      - Tue, 02 Nov 2021 10:40:40 GMT
+      - Wed, 09 Feb 2022 14:02:07 GMT
       x-ms-return-client-request-id:
       - 'true'
     method: POST
-    uri: https://sanitized.communication.azure.com/teamsUser/:exchangeAccessToken?api-version=2021-10-31-preview
+    uri: https://sanitized.communication.azure.com/teamsUser/:getToken?api-version=2022-06-01
```

Comment:
`getToken` Seems that we need to regenerate the rest client using autorest and re-record the mocks

Reply:
or maybe this is a leftover file that can be deleted?

Reply:
yeah, they were leftovers, deleted